examples/platforms/simulation/trel.c

```diff
@@ -366,8 +366,8 @@ void otPlatTrelRegisterService(otInstance *aInstance, uint16_t aPort, const uint
 #endif
 }
 
-void otPlatTrelSend(otInstance *      aInstance,
-                    const uint8_t *   aUdpPayload,
+void otPlatTrelSend(otInstance       *aInstance,
+                    const uint8_t    *aUdpPayload,
```

Comment:
Is there a way to keep `*` same as before (to reduce the `diff`)?

Reply:
I found `PointerAlignment` and maybe setting it to `Left` can work?

Reply:
Setting `PointerAlignment=PAS_Left` would remove the space between type and `*`. For example:
```
void otPlatTrelSend(otInstance*       aInstance,
                    const uint8_t*    aUdpPayload,
```

Reply:
I guess we end up with a `diff` for all case.  I would say the right alignment `otInstance       *aInstance` looks nicer than `otInstance*` . 

There seems to be a `PAS_Middle` option as well.

Reply:
With `Middle` do we still get `diff` for the multi-line case (I guess we may if it tries to add space on both sides of `*` or `&`).

I think no matter what we end up with some big `diff`. Looks-wise I like the right alignment `otInstance  *aInstance` over other options. 😃

Reply:
Thanks. `Right` has the smallest `diff` and looks better than others (in my view) :)